packages/shared/src/models/appStringEvaluator/appStringEvaluator.spec.ts

```diff
@@ -16,4 +16,11 @@ describe("App String Evaluator", () => {
       "Hello Ada Lovelace"
     );
   });
+
+  it("Evaluates boolean strings as booleans", () => {
+    expect(evaluator.evaluate("false")).toEqual(false);
+    expect(evaluator.evaluate("FALSE")).toEqual(false);
+    expect(evaluator.evaluate("true")).toEqual(true);
+    expect(evaluator.evaluate("TRUE")).toEqual(true);
```

Comment:
Should this also apply to "True" and "False"? Maybe first convert string to lower case and then compare against "true" / "false" ?

Reply:
Good spot. The actual logic that checks the string uses a regex to match strings of any case (see the `booleanStringToBoolean()` function [here](https://github.com/IDEMSInternational/parenting-app-ui/blob/master/packages/shared/src/utils/string-utils.ts)). So this is just an issue with my test cases not being extensive, I can update them.

Reply:
https://regexr.com/ is a brilliant tool for testing and understanding regex expressions, if you're not aware
<img width="200" alt="Screenshot 2024-04-30 at 17 26 59" src="https://github.com/IDEMSInternational/parenting-app-ui/assets/64838927/66dd22f6-59b9-4056-82c3-5c81383d30ae">

Reply:
Ah sure, that makes sense - I didn't realise this change was describing test cases only :)